timescale_vector/client.py

```diff
@@ -805,7 +820,7 @@ def __init__(
             The type of the id column. Can be either 'UUID' or 'TEXT'.
         """
         self.builder = QueryBuilder(
-            table_name, num_dimensions, distance_type, id_type, time_partition_interval, infer_filters)
+            table_name, num_dimensions, distance_type, id_type, time_partition_interval, infer_filters, schema_name)
```

Comment:
Need to add `schema_name` to the docs. Would it make it easier if instead of `None` we default to "public"? That way we don't have to do the `if self.schema_name is not None` checks.

Reply:
None and public are not the same since public is not always the default schema. None means use the default schema defined by the db